python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -26,8 +26,9 @@
 
 
 class OpenVINOModel:
-    def __init__(self, ie_network: str):
+    def __init__(self, ie_network: str, device='CPU'):
         self._ie = Core()
+        self._device = device
```

Comment:
how many devices do we support? You should make sure that, this device is valid.

Reply:
OpenVINO itself will make sure of that. Is that necessary for us to check this again? 
Supported devices https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_Supported_Devices.html